### PR TITLE
Add blender 2.8 docset

### PR DESCRIPTION
**Pull request by ricksilliker**
**What does this PR do?**

- Just adds the latest version of the Python API docs for Blender. Version 2.8 came out recently.

Built following the README in the Blender folder.

### CODE REVIEW

**Comment by Kapeli** [comment]
You didn't update the [main version](https://github.com/Kapeli/Dash-User-Contributions/blob/a49097d478d9fe66885b9c16e4bd4f375afa3835/docsets/Blender/docset.json#L3). Is this intentional?

**Comment by ricksilliker** [comment]
Nope, I just derped, and missed that. whoops.

File: docsets/Blender/docset.json [hunk 1]
```diff
@@ -1,12 +1,16 @@
 {
     "name": "Blender",
-    "version": "2.79",
+    "version": "2.80",
     "archive": "Blender.tgz",
```
Comment (Kapeli) [inline docsets/Blender/docset.json]: You need to update the main archive as well.

**Comment by Kapeli** [comment]
Thank you! 👍 

Please note that you need to add the actual main archive (the one with the path listed where my comment was). I'll fix it.

**Pull request merged**